docs/right-click-menu.md

```diff
@@ -0,0 +1,102 @@
+|Page action options|These options help the user manage the different Pages in the story|
+
+### Actions
+
+Different actions will be rendered depending on the Element that is right clicked. The actions will do the following:
+
+#### Page element
+
+|Action text|Action description|
+|--|--|
+|Copy|Copies the page styles from the current page to the clipboard. This does not create a new page.|
+|Paste|Pastes the page styles from the clipboard to the current page.|
+|Delete|Deletes the current page (if the page can be deleted).|
+|Add new page after|Adds a page after the current page.|
+|Add new page before|Adds a page before the current page.|
+|Duplicate page|Creates a new page that is identical to the current page. All animations, styles, and elements should be the same.|
+|Delete page|Deletes the current page (if the page can be deleted).|
+
+#### Text element
+
+|Action text|Action description|
+|--|--|
+|Copy|Copies the text and the text styles to the clipboard.|
+|Paste|Pastes the text and the text styles from the clipboard. This creates a new text element.|
+|Delete|Deletes the currently selected text element.|
+|Send to back|Set the text element behind all other elements on the page.|
+|Send backward|Bring the text element backward one layer.|
+|Bring forward|Bring the text element forward one layer.|
+|Bring to front|Set the text element in front of all elements on the page.|
+|Copy style|The styles of the selected text element are copied to the clipboard. A snackbar is displayed on completion.|
+|Paste style|The styles that are saved to the clipboard are pasted to the currently selected text box. This does not update the text in the textbox. A snackbar is shown on completion.|
+|Add style to "Saved styles"|The style of the currently selected textbox is saved to the "Saved styles" panel in the inspector. This action opens the design tab, opens the "Saved styles" panel, and collapses all other panels in the inspector. The "Saved styles" panel should be highlighted when opened.|
+|Add color to "Saved colors"|The color(s) of the currently selected textbox is saved to the "Saved colors" panel in the inspector. This action opens the design tab, opens the "Saved colors" panel, and collapses all other panels in the inspector. The "Saved colors" panel should be highlighted when opened. A snackbar is displayed on completion.|
+
+#### Background Media (images, gifs, videos) element
+
+|Action text|Action description|
+|--|--|
+|Copy|Copies the media and styles that are set to be the background to the clipboard.|
+|Paste|Pastes the media and styles from the clipboard to be the page background. This does not create a new element.|
+|Delete|Deletes the background media and styles.|
+|Detach image from background|Removes the media from the background of the page and sets it in the foreground.|
+|Replace background image|TBD (should the button only highlight the library, or prompt the user to replace it, how would this work, etc.)|
```

Comment:
good question, i asked about this and I was told that it should open the dialog to replace the image but I'm not super sure about that because what if the user is relying on 3p media? Then it should go to that tab. Regardless, this and the quick action menu should probably have consistency. Thanks for calling it out as a TBD/TODO.